mkdocs.yml

```diff
@@ -12,7 +12,7 @@ nav:
     - Local Development Environment: 'self-hosted-nmaas/local-vm.md'
   - NMaaS Applications:
     - List of Supported Applications: 'nmaas-applications/application-list.md'
-    - Adding a new Application: 'nmaas-applications/new-application.md'
+    - Adding a new Application: 'nmaas-applications/new-application/new-application.md'
```

Comment:
Please move the markdown file one folder up, so that the final path is `nmaas-applications/new-application.md`. No need to put it into a folder by itself.